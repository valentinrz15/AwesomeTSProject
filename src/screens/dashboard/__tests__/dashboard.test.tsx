import React from 'react';
import renderer from 'react-test-renderer';
import { Dashboard } from '../index';

describe('dashboard test', () => {
  it('should render dashboard snapshot', () => {
    const { toJSON } = renderer.create(<Dashboard />);

    expect(toJSON()).toMatchSnapshot();
  });
});
