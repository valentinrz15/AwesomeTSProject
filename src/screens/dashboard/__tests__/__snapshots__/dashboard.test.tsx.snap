// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard test should render dashboard snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFF",
      "flex": 1,
    }
  }
  testID="dashboard-index-container"
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFF",
        "justifyContent": "center",
      }
    }
    testID="dashboard-index-content"
  >
    <Text>
      Welcome to Lokalise!
    </Text>
    <Text>
      https://myapi.preprod.com
    </Text>
    <Text>
      Esto es una prueba
    </Text>
  </View>
</View>
`;
